modules/oe_content_call_proposals/oe_content_call_proposals.info.yml

```diff
@@ -0,0 +1,41 @@
+name: OpenEuropa Call for proposals Content
+description: Contains the EC Call for proposals content type.
+package: OpenEuropa
+type: module
+core: 8.x
+
+dependencies:
+  - link:link
+  - datetime:datetime
+  - oe_content:oe_content_entity_contact
+  - oe_content:oe_content_reference_code_field
+  - oe_content:oe_content_departments_field
+  - oe_content:oe_content_documents_field
+
+config_devel:
+  install:
```

Comment:
Could you please add field "Introduction" (oe_summary). This field isn't specified in spec but we use it in the header. 
See example: build/modules/contrib/oe_content/modules/oe_content_call_tenders/config/install/field.field.node.oe_call_tenders.oe_summary.yml (the same settings can be used for Call for proposals).

<img width="1039" alt="Screenshot 2020-11-04 at 18 22 15" src="https://user-images.githubusercontent.com/1183951/98129953-c60e8700-1eca-11eb-96c4-928466b37a42.png">

Reply:
I added the Introduction field.